common/viewer.cpp

```diff
@@ -108,8 +108,6 @@ namespace rs2
                 ImGui::PushStyleColor(ImGuiCol_Text, grey);
                 ImGui::Text("Polygon File Format defines a flexible systematic scheme for storing 3D data");
                 ImGui::PopStyleColor();
-                ImGui::NewLine();
```

Comment:
`ImGui::SetCursorScreenPos({ (float)(x0 + 15), (float)(y0 + 90) });`
set position and it looks unreadable with a font of 20.
![image](https://github.com/IntelRealSense/librealsense/assets/26552140/e2f2453a-0f32-41d8-b251-ca2d25b065e6)


How looks window **without** `ImGui::SetCursorScreenPos({ (float)(x0 + 15), (float)(y0 + 90) });`
![image](https://github.com/IntelRealSense/librealsense/assets/26552140/d234616b-c7f1-450f-8187-b3eb03a891fe)

`NewLine()` I can return because it looks more readable with it.

Reply:
OK, so with NewLine it will look better?

Reply:
Yes. I will leave it with a `NewLine`